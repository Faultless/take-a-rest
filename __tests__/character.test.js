const axios = require('../__mocks__/axios');

describe('CRUD operations for a single character', () => {
  it('should fetch a user by his ID', () => {
    axios.get('example/api/character/1').then(res => expect(res).toMatchSnapshot());
  });
  it('should fail to fetch a user by his ID', () => {
    axios.get('example/api/character/2').catch(err => expect(err).toMatchSnapshot());
  });
});
