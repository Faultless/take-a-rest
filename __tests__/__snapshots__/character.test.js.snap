// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CRUD operations for a single character should fail to fetch a user by his ID 1`] = `
Object {
  "error": "Bad Request.",
}
`;

exports[`CRUD operations for a single character should fetch a user by his ID 1`] = `
Object {
  "description": "",
  "id": 1,
  "image": "http://i.annihil.us/u/prod/marvel/i/mg/c/e0/535fecbbb9784.jpg?apikey=498749143bba8e7a297a3589548ed657",
  "name": "3-D Man",
  "rating": 4,
}
`;
